docs/how_to_guide/custom_predictors.md

```diff
@@ -0,0 +1,61 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(custom-features)=
+# Custom Features
```

Comment:
I don't think "custom" is the right word here, but I don't have a better one.

Reply:
Precomputed?